src/Symfony/Component/HttpFoundation/IpUtils.php

```diff
@@ -64,9 +64,13 @@ public static function checkIp4($requestIp, $ip)
         if (false !== strpos($ip, '/')) {
             list($address, $netmask) = explode('/', $ip, 2);
 
-            if ($netmask < 1 || $netmask > 32) {
+            if ($netmask < 0 || $netmask > 32) {
                 return false;
             }
+
+            if ('0' === $netmask) {
```

Comment:
Sorry, cast is not the word :) i say: `0 === $netmask` instead of `'0' === $netmask`